bootstrap/pkg/kfapp/coordinator/coordinator.go

```diff
@@ -299,6 +303,9 @@ func NewKfApp(options map[string]interface{}) (kftypes.KfApp, error) {
 	kfDef.Spec.UseBasicAuth = useBasicAuth
 	kfDef.Spec.UseIstio = useIstio
 	kfDef.Spec.PackageManager = packageManager
+	kfDef.Spec.EnableApplications = true
```

Comment:
dup with line 278
BTW, do this mean `kfDef.Spec.EnableApplications` is always true? if yes, I think we may not need this flag and just always include `application` into overlays